vnext/JSI/Universal/ChakraJsiRuntime_edgemode.cpp

```diff
@@ -87,13 +63,15 @@ facebook::jsi::Value ChakraRuntime::evaluateJavaScriptSimple(
     throw facebook::jsi::JSINativeException("Script URL can't be empty.");
 
   JsValueRef result;
-  checkException(JsRunScript(
-      script16.c_str(),
-      JS_SOURCE_CONTEXT_NONE /*sourceContext*/,
-      url16.c_str(),
-      &result));
-
-  return createValue(result);
+  ThrowUponJsError(
+      JsRunScript(
+          script16.c_str(),
+          JS_SOURCE_CONTEXT_NONE /*sourceContext*/,
+          url16.c_str(),
+          &result),
+      "JsRunScript");
```

Comment:
>"JsRunScript" [](start = 6, length = 13)

Does this message give any valuable info? Such string messages are error prone, imagine someone refactors this Api, and forgets to update this string. We should be able to create similar text from a macro, right?

Reply:
On second thought, having this string here is pretty useless so I'll remove it.

---
In reply to: [336148449](https://github.com/microsoft/react-native-windows/pull/3445#discussion_r336148449) [](ancestors = 336148449)